content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Install Istio Multicluster on Google Kubernetes Engine clusters
+description: Example multicluster GKE install of Istio.
+weight: 65
+keywords: [kubernetes,multicluster]
+---
+
+This is an example deployment of multicluster Istio on 2
```

Comment:
This paragraph can be shortened:

This example demonstrates how to use Istio's multicluster feature to join 2 Google Kubernetes Engine clusters together, using the installation instructions.

Reply:
ACK ... including links ok?